src/main/java/com/ericsson/ec/JenkinsManager.java

```diff
@@ -105,25 +104,23 @@ public boolean createJob(String jobName, String jobXmlData) throws Exception {
             throw new Exception("A job is no one! Jenkins do not like no one!");
         }
 
-        httpRequest.setBaseUrl(jenkinsBaseUrl).addHeader("Authorization", "Basic " + encoding)
-                .addHeader("Content-type", MediaType.APPLICATION_XML).addHeader("Jenkins-Crumb", crumb)
-                .addParam("name", jobName).setBody(jobXmlData).setEndpoint("/createItem");
+        httpRequest.setBaseUrl(jenkinsBaseUrl).addHeader("Authorization", "Basic " + encoding).addHeader("Content-type", MediaType.APPLICATION_XML)
```

Comment:
Chains these calls under eachother, easier to see whats happening and also shorter lines :)

Reply:
Done.